### ISSUE TITLE
Pplayer controls not WCAG 2.x compliant and lacking keyboard navigation

### ISSUE THREAD

**Issue by rcoenen** [LottieFiles/lottie-player#206]
**Description:**
The Lottie web player controls for auto-started looping animations are not WCAG 2.x compliant. According to current WCAG 2.1 and WCAG 2.2 (draft) guidelines, animations that auto-start and loop must have controls that allow users to pause, stop, or hide the animation. You can read more about this guideline on the following link: https://www.w3.org/WAI/WCAG22/Understanding/pause-stop-hide.html.

Currently, the Lottie web player controls for auto-started looping animations only include a progress bar and a play/pause button. This means that users who rely on keyboard navigation cannot control the animation or pause it if it is causing a problem.

**Proposal:**
To ensure that the Lottie web player controls for auto-started looping animations are WCAG 2.x compliant, I propose the following solutions:

1. Make the controls keyboard navigable: This will allow users who rely on keyboard navigation to control the animation and pause it if necessary.
2. Provide minimum controls: Consider providing minimum controls, such as play/pause button, and hide animation button, without a progress bar. This will ensure that users can pause and hide the animation if necessary, without being distracted by a moving progress bar.
3. Make minimum play controls CSS stylable: Consider making the minimum play controls CSS stylable so that stakeholders can customize the controls to match their brand guidelines or accessibility requirements.
4. Automatically show the "pause" button once the animation has played for 5 seconds: This will provide users with a clear indication that they can pause the animation if necessary, without cluttering the controls with unnecessary buttons.
5. Implementing all of these solutions would greatly improve the accessibility of the Lottie web player for users with disabilities. Specifically, making the controls keyboard navigable is critical for users who cannot use a mouse or trackpad to interact with the player.

Thank you.

**Comment by rcoenen** [comment]
Here is an example of the logical outcome of WCAG 2.1 on looped Lottie animations. It seems a little absurd, and I doubt this is the true intention of WCAG. 

https://codepen.io/rcoenen/pen/xxyxzZV

**Comment by samuelOsborne** [comment]
Hi again @rcoenen, Im just going to post here for transparency on the solution we discussed on slack. 

The steps of the solution are :

 - checking for `@media (prefers-reduced-motion: reduce)` 
 - if we detect this preference, disable auto play and add controls (or just a play / pause button) so that the user can choose to see the animation or not

**Comment by gleatherman** [comment]
is there a timeline on implementing the reduced motion flag to disable auto-play?
